taqueria-vscode-extension-web-ui/src/data-editors/BoolEditor.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export const BoolEditor = (
+	{ value, onChange }: { value: Record<string, any>; onChange: (value: Record<string, any>) => void },
+) => {
+	if (value === null || value === undefined || typeof value !== 'object') {
```

Comment:
This evaluation can be simplified to just `typeof value !== 'object'`

Reply:
But: `typeof null === "object";`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof#typeof_null

I will probably make a helper function for these